fdb-record-layer-lucene/src/test/java/com/apple/foundationdb/record/lucene/LuceneIndexMaintenanceTest.java

```diff
@@ -1099,6 +1099,59 @@ void randomlyRemoveAllRecords() throws IOException {
                     assertThat(partitionCounts, contains(0)));
     }
 
+    static Stream<Arguments> multiUpdate() {
+        return ParameterizedTestUtils.cartesianProduct(
+                ParameterizedTestUtils.booleans("isSynthetic"),
+                ParameterizedTestUtils.booleans("isGrouped"),
+                Stream.of(0, 10),
+                Stream.of(0, 1, 4),
+                Stream.of(5365));
```

Comment:
Should this use `RandomizedTestUtils.randomizedSeeds` to run extra seeds during nightly?